src/bulma/index.scss

```diff
@@ -7,10 +7,6 @@ $primary: $theme-red;
 $link: $grey-dark;
 $link-hover: $black-bis;
 
-a:hover {
-  text-decoration: underline;
```

Comment:
Is this rule applying for all hovered anchors?

Reply:
Actually, yes but as our new design doesn't require underline on hover no more, so I have removed.